redash/query_runner/uptycs.py

```diff
@@ -0,0 +1,141 @@
+import os
+import sys
+import json
+import jwt
+import datetime
+import requests
+import urllib3
+import logging
+logger = logging.getLogger(__name__)
+
+from redash.query_runner import *
+from redash.utils import JSONEncoder
+
+enabled = True
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+class Uptycs(BaseSQLQueryRunner):
+
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "url": {
+                    "type": "string"
+                },
+                 "customerid": {
+                    "type": "string"
+                },
+                "key": {
+                    "type": "string"
+                },
+                "secret": {
+                    "type": "string",
+                },
+            },
+            "order": ['url', 'customerid', 'key', 'secret'],
+            "required": ["url", "customerid", "key","secret"],
+            "secret": ["secret", "customerid", "key"]
```

Comment:
Are you sure customer id and key should be secret too?

Reply:
Yes. This is intentional. We want to keep it as secret.

Reply:
Why?

Reply:
This is purely based on Uptycs way of keeping combinations secret. However, will check with the team one more time.

Reply:
I think that atleast having one of them as non secret is better, because otherwise the admin has no way of knowing what configuration this is about. Values marked as secret are never sent back to the UI.

Also note that the other values are only available to the admin.

Reply:
Based on the discussion, removed customer_id from secret